src/client/main.cpp

```diff
@@ -211,9 +211,9 @@ int main(int argc, const char **argv)
 		tempo::addr_r = argv[1];
 	if (tempo::addr_r == "127.0.0.1") {
 		std::srand(time(NULL));
-		int d          = std::rand() % 10;
-		tempo::port_ci = DEFAULT_PORT_IN + 10 + d;
-		tempo::port_co = DEFAULT_PORT_OUT + 10 + d;
+		int d          = std::rand() % 1000;
```

Comment:
Technically an undocumented change, but I'm not complaining

Reply:
Snakes! This is for your benefit :stuck_out_tongue:

Reply:
This doesn't break anything does it? If it selects a port which is already being used for example.

Reply:
Then it will not be able to open sockets. But there's a reasonably low likelihood of that occurring.